tools/version.c

```diff
@@ -30,7 +30,6 @@ void version ()
 	printf (RELEASE " ");
 	printf ("ISO C ");
 	printf (COMPANY " ");
-	printf (COMPILE "\n");
```

Comment:
The line-break is missing after your patch. Maybe you can just add this to the company line ?